api/api.py

```diff
@@ -1,4 +1,6 @@
-"""Book scraping api"""
+"""
+Book scraping api.This api scrape paperbackswap website to get book request details
+"""
```

Comment:
Book scraping api. This api scrape paperbackswap website to get book request details.